### DISCUSSION TITLE
this project is abandoned?

### DISCUSSION THREAD

**Discussion by marcellovocale** [lllyasviel/Fooocus#3721] [Q&A] [answered]
I'm encountering several issues and errors on the platform, and I’ve noticed that customer support hasn’t responded for days. Does anyone know if the project has been abandoned or if any updates are coming? I’d like to understand if it's worth continuing to invest time and resources or if it’s better to consider other options. Any information would be appreciated. Thank you!

**Comment by DavidDragonsage** [comment]
I don't understand what you mean by customer support - are you using an online service? That has nothing to do with Fooocus itself. Anyone can take this project and use it to make money. The Fooocus UI is not affiliated with any online services.

There are so minor issues with Fooocus itself, but nothing dramatic. It sounds like the online service, if that is what you are using, is where the problem lies.

No software is perfect, but Fooocus is at the stage where continuous bug fixes are not required. This is the statement on support from the developer:

"Project Status: Limited Long-Term Support (LTS) with Bug Fixes Only

"The Fooocus project, built entirely on the Stable Diffusion XL architecture, is now in a state of limited long-term support (LTS) with bug fixes only. As the existing functionalities are considered as nearly free of programmatic issues (Thanks to [mashb1t](https://github.com/mashb1t)'s huge efforts), future updates will focus exclusively on addressing any bugs that may arise.

"There are no current plans to migrate to or incorporate newer model architectures. However, this may change during time with the development of open-source community. For example, if the community converge to one single dominant method for image generation (which may really happen in half or one years given the current status), Fooocus may also migrate to that exact method."

There are two active branches of Fooocus that have gone on to support Flux and other new models,
SimpleSDXL2: https://github.com/metercai/SimpleSDXL
RuinedFooocus: https://github.com/runew0lf/RuinedFooocus

If you ever need help installing SimpleSDXL2 we have resources at https://www.facebook.com/groups/fooocus/

**Reply by marcellovocale** [reply]
Ah, okay, I was asking because I noticed results that were very different from what I expected. For example, I would write 'Make a woman with blue eyes,' and it would return a woman with dark eyes. This is just one example, but I could give you a hundred more like it.

Also, I noticed in the issue section that many technical problem questions had no response even after several weeks, so I started to wonder if the project had been abandoned.

Thanks for the response; I’ll continue using Foocus.

**Reply by DavidDragonsage** [reply, marked as answer]
@marcellovocale  Yes, I admit that the "Long-Term Support" policy does not seem to be a reality, which is a bit frustrating!

But the problem of getting dark eyes when you asked for blue is more likely a problem with the base model you are using. Some are a lot more faithful to prompts than others.

Also, check what styles you have selected. The styles are really just prompt extensions that add extra words to the prompt. For example, if you select "Dark Fantasy" it may increase the chance of dark eyes (I'm just using a hypothetical example, I have not actually tested if this is the case with this style). 

My tests at Pure Fooocus have found the Cheyenne 1.8 and JuggernautXL XI are particularly good at prompt following in reference to facial appearance. But there are always tradeoffs with base models, with some having more strengths in certain areas, offset by weaknesses in others. Currently the overall winner out of the 24 base models tested, including Flux variants, is SDXL FaeTastic 2.4.
